dashboard/app/models/skill.rb

```diff
@@ -8,9 +8,42 @@
 #  concept             :string(255)
 #  created_at          :datetime         not null
 #  updated_at          :datetime         not null
+#  key                 :string(255)      not null
+#
+# Indexes
+#
+#  index_skills_on_key  (key) UNIQUE
 #
 class Skill < ApplicationRecord
   validates :description, presence: true
 
   has_and_belongs_to_many :levels, join_table: 'levels_skills'
+
+  def seeding_key(seed_context)
+    {'skill.key': key}.stringify_keys
+  end
+
+  def self.setup
+    sample_skills = [
+      {
+        description: "Declare variables correctly",
+        evaluation_criteria: "Did the students declare all of the variables in their code correctly?",
+        concept: "Variables"
+      },
+      {
+        description: "Name variables according to conventions",
+        evaluation_criteria: "Are there any spaces in variable names? Are there any misspelled variable names? Do variable names follow casing conventions?",
+        concept: "Variables"
+      },
+      {
+        description: "Increment values stored in variables",
+        evaluation_criteria: "Does the student's added code increment the values stored in the variables correctly?",
+        concept: "Variables"
+      },
+    ]
+    transaction do
+      reset_db
```

Comment:
can you explain how this line is working? it looks like you need to `include Seeded` to load the definition from here: https://github.com/code-dot-org/code-dot-org/blob/45452cf683d2c013a1ad7af33dc7aa619b2b90a5/dashboard/app/models/concerns/seeded.rb#L7